activerecord/test/cases/connection_adapters/merge_and_resolve_default_url_config_test.rb

```diff
@@ -9,12 +9,14 @@ def setup
         @previous_database_url = ENV.delete("DATABASE_URL")
         @previous_rack_env = ENV.delete("RACK_ENV")
         @previous_rails_env = ENV.delete("RAILS_ENV")
+        @adapters_was = ActiveRecord::ConnectionAdapters.instance_variable_get(:@adapters).dup
```

Comment:
This is pretty bad. But there's no real argument to expose a way to read/write this that isn't in a specific testing scenario I guess.